run.sh

```diff
@@ -66,7 +66,7 @@ function delete_on_kvm(){
 function clean_k8s(){
     echo "DANGER ZONE"
     read -p "Press Enter or Ctrl-C. Enter will remove all the deployed components on your kubernetes cluster"
-    ansible -m script -a "script_library/cleanup-k8s.sh" localhost
+    ansible -m script -a "script_library/cleanup-k8s.sh" osh-deployer -i inventory-osh.ini
```

Comment:
Is it correct syntax to limit execution to osh-deployer ? I recall using --limit or -l as option to limit against specific group/host.

Reply:
it is. You could limit, but you'd still need to pass a group.

For example
```
ansible -m script -a "script_library/cleanup-k8s.sh" all -i inventory-osh.ini --limit osh-deployer
```

It makes more sense in terms of ansible workers/strategy to target the right nodes directly.